src/main/resources/application.yaml

```diff
@@ -69,9 +76,8 @@ jackson:
 azure:
   service-bus:
     hmc-to-hearings-api:
-      connectionString: ${HEARINGS_TOPIC_CONNECTION_STRING:}
-      topicName: ${HEARINGS_TOPIC_NAME:}
-      subscriptionName: ${HEARINGS_TOPIC_SUBSCRIPTION_NAME:}
+      topicName: ${HEARINGS_TOPIC_NAME:lucas-topic}
```

Comment:
optional: change to local-topic? do you really want your name attached to this? :p